src/notation/view/notationpaintview.cpp

```diff
@@ -876,16 +876,20 @@ bool NotationPaintView::event(QEvent* ev)
     return QQuickPaintedItem::event(ev);
 }
 
-bool NotationPaintView::eventFilter(QObject* obj, QEvent* ev)
+bool NotationPaintView::eventFilter(QObject* object, QEvent* event)
```

Comment:
Since this event filter only filters events for the NotationPaintView itself, wouldn't it make more sense to convert it to [`bool NotationPaintView::event(QEvent* event)`](https://doc.qt.io/qt-5/qobject.html#event) and remove `installEventFilter(this);` in the constructor?
It seems that the body of the function can stay unchanged, except that you need `return QQuickPaintedItem::event(event);` at the end.

Reply:
thanks, fixed